src/Microsoft.OData.Core/UriParser/Parsers/OpenTypeMethods.cs

```diff
@@ -1,731 +0,0 @@
-﻿//---------------------------------------------------------------------
-// <copyright file="OpenTypeMethods.cs" company="Microsoft">
-//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
-// </copyright>
-//---------------------------------------------------------------------
-
-namespace Microsoft.OData.UriParser
-{
-    #region Namespaces
-    using System;
-    using System.Collections.Generic;
-    using System.Diagnostics;
-    using System.Diagnostics.CodeAnalysis;
-    using System.Linq.Expressions;
-    using System.Reflection;
-    using Edm = Microsoft.OData.Edm;
-    #endregion
-
-    /// <summary>Use this class to perform late-bound operations on open properties.</summary>
-    /// <remarks>This class was copied from the product.</remarks>
-    internal static class OpenTypeMethods

```

Comment:
>OpenTypeMethods [](start = 26, length = 15)

can you include a comment in the commit/PR description as to why it's OK to delete this code?